actions/update-argocd/action.yaml

```diff
@@ -0,0 +1,163 @@
+name: Update argocd-clusters
+description: Update solution or data value in argocd cluster repo
+
+inputs:
+  gh_token:
+    description: Token for GH argo repository
+    required: true
+  target:
+    description: Specify "develop" or "stage" or "testnet"
+    required: true
```

Comment:
@marchuk-vlad , maybe use
```yaml
        type: choice
        options:
          - stage
          - develop
          - testnet
```
will be better

Reply:
There is no types in inputs in [composit actions](https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action). We can specify only  `description` `required` `default`.